src/annotator/guest.js

```diff
@@ -478,8 +463,8 @@ export default class Guest extends Delegator {
       // Add the anchors for this annotation to instance storage.
       this._updateAnchors(this.anchors.concat(anchors));
 
-      // Let plugins know about the new information.
-      this.plugins.CrossFrame?.sync([annotation]);
+      // Let the sidebar know about the new annotation.
```

Comment:
This comment could be more general, isn't? Now that we have the notebook, which is in another iframe, it could communicate with that as well.

Reply:
Yes, that's true.